dev/PushNotifications/PushNotificationsLongRunningTask/platform.cpp

```diff
@@ -133,6 +143,28 @@ STDMETHODIMP_(HRESULT __stdcall) NotificationsLongRunningPlatformImpl::Unregiste
 }
 CATCH_RETURN()
 
+STDMETHODIMP_(HRESULT __stdcall) NotificationsLongRunningPlatformImpl::AddToastRegistration(_In_ PCWSTR processName, _In_ PCWSTR toastGuid) noexcept try
```

Comment:
Please remove inferences to toastGuid everywhere.

Reply:
This has been changed in the latest commit, not sure why it's not being reflected here yet.